scenarios/centos-9/edpm_ci.yml

```diff
@@ -13,13 +13,11 @@ cifmw_edpm_deploy_run_validation: true
 cifmw_edpm_prepare_skip_crc_storage_creation: true
 cifmw_edpm_prepare_skip_openstack_operator: true
 cifmw_edpm_prepare_make_openstack_deploy_env:
-  OUT: "{{ cifmw_installyamls_repos }}/out"
```

Comment:
In Zuul CI, We run `make openstack` in the pre-run step. We generate the CR/manifests in "{{ cifmw_installyamls_repos }}/out" directory where cifmw_installyamls_repos is defined here.
```
cifmw_installyamls_repos: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/install_yamls"
```
Since we are removing out dir. Can we set the cifmw_edpm_prepare_manifests_dir to {{ cifmw_installyamls_repos }}/out ?

The jobs are passing without it. But It will help to keep the manifest at one place.

Reply:
Based on discussion with pablo, we need to store all the generated contents in manifest dir. Since jobs are passing. Let's merge it.